library/stdtypes.po

```diff
@@ -4859,61 +4862,57 @@ msgstr ""
 "représenté lors de l’affichage ::"
 
 #: ../Doc/library/stdtypes.rst:2989
-#, fuzzy
 msgid ""
 "Return ``True`` if all bytes in the sequence are alphabetical ASCII "
 "characters or ASCII decimal digits and the sequence is not empty, ``False`` "
 "otherwise. Alphabetic ASCII characters are those byte values in the sequence "
 "``b'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'``. ASCII decimal "
 "digits are those byte values in the sequence ``b'0123456789'``."
 msgstr ""
-"Donne ``True`` si tous les caractères de la chaîne sont des caractères ASCII "
-"alphabétiques ou chiffres. et que la séquence n'est pas vide, sinon "
-"``False``. Les caractères ASCII alphabétiques sont les suivants "
+"Renvoie ``True`` si tous les caractères de la chaîne sont des caractères "
+"ASCII alphabétiques ou chiffres. et que la séquence n'est pas vide, sinon "
+"``False``. Les caractères ASCII alphabétiques sont les suivants dans la "
+"séquence de octets "
 "``b'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'`` et les "
 "chiffres : ``b'0123456789'``."
```

Comment:
```suggestion
"chiffres : ``b'0123456789'``."
```

 /tmp/padpo_l4zwefq9/library/stdtypes.po:4864 [     NBSP     ] There should be a non-breakable space before ":": between ###PQRSTUVWXYZ' » et les chiffres ### and ###: « b'0123456789' ».###